src/components/ImageAudio/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function ImageAudio() {
+    return(
+        <img className="image" src="https://via.placeholder.com/80"></img>
+    )
+};
```

Comment:
Me parece que el resto de archivos tiene tabs de 2 espacios.
```suggestion
function ImageAudio() {
    return(
        <img className="image" src="https://via.placeholder.com/80" /> 
    )
};
```

También, recomendaría que la URL de la imagen viniese por props, con eso pueden reutilizarla desde cualquier lugar. Y en caso de que no llegue una imagen para ese audio, podrías usar esa URL que tienes ahí como [default props](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values).

Reply:
corregí los espaciados, lo de la url (la imagen) provendría de la Api en realidad, pero igualmente realice default props

Reply:
💎